### PR TITLE
Code clean up based on Google style and eslint rules

### PR DESCRIPTION
**Pull request by zhx828**

### CODE REVIEW

File: app/scripts/controller/sync.js [hunk 1]
```diff
@@ -1,67 +1,33 @@
...
 /**
  * @author suny1@mskcc.org on 3/15/16.
  */
 
 'use strict';
-(function(iViz, $, _){
+(function(iViz, $, _) {
   iViz.sync = {};
   // syncing util: select samples or patients based on only samples/patients filters
   iViz.sync.selectByFilters = function(filters, data) { // type: sample or patient
```
Comment (kalletlak) [inline app/scripts/controller/sync.js]: I think this function is not used anymore. We can get rid of it.

File: app/scripts/controller/sync.js [hunk 2]
```diff
@@ -74,12 +40,12 @@
   iViz.sync.selectByCases = function(type_, data_, cases_) {
```
Comment (kalletlak) [inline app/scripts/controller/sync.js]: Even this function is not used anymore. We can get rid of this complete file.

**Comment by zhx828** [comment]
Issues not addressed in the code reformat:
1. main.js _data doesn't have any reference
2. line 276 has http://eslint.org/docs/rules/no-loop-func issue
3. Issues with proxy.js under survivalChart folder

**Comment by zhx828** [comment]
@yichaoS Could you take a look all survival plots related files? Are we going to use these files in iViz or in cBioPortal?

**Pull request merged**